core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -1228,6 +1229,7 @@ private void startIntentBasedOnAction(Intent intent) {
     switch (intent.getAction()) {
       case Intent.ACTION_PROCESS_TEXT: {
         goToSearchWithText(intent);
+        removeIntentAction(intent);// see https://github.com/kiwix/kiwix-android/issues/2607
```

Comment:
Do we really need that comment?

Reply:
If you want then I can remove it :)

Reply:
If you want to keep it there should a space before the comment.